src/coreclr/jit/codegenarm64.cpp

```diff
@@ -733,7 +730,7 @@ void CodeGen::genBuildRegPairsStack(regMaskTP regsMask, ArrayStack<RegPair>* reg
                     {
                         isPairSave = true;
 
-                        regsMask &= ~reg2Mask;
+                        regsMask &= ~genRegMask(reg2);
```

Comment:
Should this one be `regMask ^= genRegMask(reg2)`

Reply:
ah, yes, forgot to update it.